locals.tf

```diff
@@ -58,6 +67,7 @@ locals {
 
   // SampleCount statistic adds 2 to the real count in case the engine is postgres, so 7 means 5 + 2
   slow_queries_alert_threshold = var.engine == "postgres" ? 7 : 5
+  parameter_group_family       = format("%s%s%s", (local.is_aurora ? "aurora-" : ""), var.engine, (var.engine == "mariadb" ? regex("\\d+\\.\\d+", var.engine_version) : var.engine_version))
```

Comment:
did we test this change with all engines?
it seems the original one differs, especially for postgresql and mariadb,
also as I understood this supposed to be used with aurora dbs also, but db_aurora module do not use this value